machine_management/creating_machinesets/creating-machineset-rhv.adoc

```diff
@@ -0,0 +1,17 @@
+[id="creating-machineset-rhv"]
+= Creating a MachineSet on {rh-virtualization}
+include::modules/common-attributes.adoc[]
+:context: creating-machineset-rhv
+
+You can create a different MachineSet to serve a specific purpose in your
```

Comment:
Lines shouldn't be hard wrapped in the middle of sentences/paragraphs. Do you mind undoing the hard wraps? An easy way to do it is to select the hard wrapped text that you want to keep in a single paragraph and press Ctrl+J (if you are using Atom).

Reply:
I don't mind doing it either way. I know that some authors intentionally use hard wrapping. I usually don't, but I copied this file from the OpenStack provider's section, and I guess this file had it already.